lib.rs

```diff
@@ -865,6 +865,8 @@ impl<A: Array> SmallVec<A> {
 
     /// Insert multiple elements at position `index`, shifting all following elements toward the
     /// back.
+    ///
+    /// Note: when the iterator panics, this can leak memory.
```

Comment:
I added this when I read that the leak was intentional; if this is still considered a bug I can also remove it again.

Reply:
Let's keep it for now, and remove it if/when the leak is fixed.